e2e/tables.spec.ts

```diff
@@ -157,48 +160,49 @@ test.describe('Tables', () => {
           .click()
       })
 
-      await test.step('create table schema', async () => {
+      await testAuth.step('create table schema', async () => {
         await addColumnsFromColumnSchemaConfig(
           userPage,
           initialColumnsSchemaConfig,
         )
         await userPage.getByRole('button', { name: 'Next' }).click()
       })
 
-      await test.step('enter table name and description', async () => {
+      await testAuth.step('enter table name and description', async () => {
         await userPage.getByLabel('Name').fill(tableName)
         await userPage.getByLabel('Description').fill(tableDescription)
         await userPage.getByRole('button', { name: 'Finish' }).click()
       })
     })
 
-    await test.step('Table has been created with expected schema', async () => {
-      await expectTablePageLoaded(userPage, tableName, tableDescription)
+    await testAuth.step(
+      'Table has been created with expected schema',
+      async () => {
+        await expectTablePageLoaded(userPage, tableName, tableDescription)
 
-      await test.step('get table entity id', async () => {
-        tableEntityId =
-          (await userPage
-            .getByRole('row')
-            .filter({ hasText: 'SynId' })
-            .getByRole('cell')
-            .nth(1)
-            .textContent()) || undefined
-        expect(tableEntityId).not.toBeUndefined()
-      })
+        await testAuth.step('get table entity id', async () => {
+          tableEntityId =
+            (await userPage
+              .getByRole('row')
+              .filter({ hasText: 'SynId' })
+              .getByRole('cell')
+              .nth(1)
+              .textContent()) || undefined
+          expect(tableEntityId).not.toBeUndefined()
+        })
 
-      await test.step('table is currently empty', async () => {
-        await expect(
-          userPage.getByRole('heading', { name: 'Items (0)' }),
-        ).toBeVisible()
-        await expect(
-          userPage.getByText('This table is currently empty'),
-        ).toBeVisible()
-      })
+        await testAuth.step('table is currently empty', async () => {
+          await expect(
+            userPage.getByRole('heading', { name: 'Items (0)' }),
+          ).toBeVisible()
+          await expect(userPage.getByText('0 total rows')).toBeVisible()
```

Comment:
Also change the expectation for an empty table in Synapse so the tables test passes -- a table with 0 rows is shown instead of text stating that "This table is currently empty": 

<img width="1377" alt="image" src="https://github.com/Sage-Bionetworks/SynapseWebClient/assets/26949006/e91c201e-da5d-47ac-9ea3-bf3310b87e80">

Reply:
Is this expected? This seems like a new bug

Reply:
We'll expect the tables test to fail until that ticket is resolved.